### PR TITLE
TE5-T93 feat: add active record association practical

### PR DESCRIPTION
**Pull request by rutikpatel20**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    @event = Event.find(params[:event_id]).comments.create("user_comment"=>params[:user_comment], "user_id" => @current_user.id)
+    flash[:errors] = "Comment Added Successfully"
+  end
+
+  def edit_comment
+    @event = Event.find(params[:id])
+  end
+  
+  def update_comment
+    # @comment = Comment.find(params[:id])
```
Comment (dhruvika29kevat) [inline app/controllers/events_controller.rb:67]: Remove commented code.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+    @user.build_address
+  end
+  
+  def create
+    @user = User.new(user_params)
+    # @user.build_address
```
Comment (dhruvika29kevat) [inline app/controllers/users_controller.rb:9]: Remove commented code from all the files

Comment (rutikpatel20) [reply app/controllers/users_controller.rb]: Okay

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (rutikpatel20) [reply app/controllers/events_controller.rb]: Okay

File: app/views/categories/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<!-- For for New or Edit -->
+<div class="center flex-column border-warning shadow" id="vantajs" style="height:100vh;">
+  <% if flash[:errors] %>
+    <% flash[:errors].each do |error| %>
+      <p class="text-danger"><%= error %></p>
+    <% end %>
+  <% end %>
+  <% # Form Started Here %>
```
Comment (dhruvika29kevat) [inline app/views/categories/_form.html.erb:8]: For comment message use html comment tags

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  # adds virtual attributes for authentication
+  has_secure_password
+  validates :email, presence: true, uniqueness: true, format: { with: /\A[^@\s]+@[^@\s]+\z/, message: 'Invalid email' }
+  has_many :enrollments
+  has_many :events, through: :enrollments
+  # has_many :enroll_events, through: :enrollments, source: :event
```
Comment (dhruvika29kevat) [inline app/models/user.rb:7]: Remove commented code

File: app/views/categories/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<%# Add New Category Form %>
```
Comment (dhruvika29kevat) [inline app/views/categories/new.html.erb:1]: Here also use html tags for comment message

File: app/views/events/_form.html.erb [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    <% flash[:errors].each do |error| %>
+      <p class="text-danger"><%= error %></p>
+    <% end %>
+  <% end %>
+  <div class="card bg-light border-warning shadow" style="width: 22rem;">
+    <div class="card-body">
+      <div class="card-header">
+        <h2 class="text-center">Event Form</h2>
+      </div>
+      <% # Form Started Here %>
```
Comment (dhruvika29kevat) [inline app/views/events/_form.html.erb:13]: Same here use html tags for comment message

File: app/views/events/show.html.erb [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+          <td>Username </td>
+          <td>Comments </td>
+          <td colspan=2 class="text-center">Likes </td>
+          <td colspan=2 class="text-center">Action </td>
+        </thead>
+        <% @event.comments.each do |comment| %>
+          <tr>
+            <td><%= comment.user.username %> </td>
+            <td><%= comment.user_comment %> </td>
+            <%# Likes on comment %>
```
Comment (dhruvika29kevat) [inline app/views/events/show.html.erb:38]: Use html comment tags for comment message

File: config/routes.rb [hunk 8]
```diff
@@ -58,10 +62,48 @@
...
+
+  # Active Record Association
+  root 'events#index'
+  get 'sign_up', to: 'users#new'
+  post 'sign_up', to: 'users#create'
+  get 'sign_in', to: 'sessions#new'
+  post 'sign_in', to: 'sessions#create', as: 'log_in'
+  delete 'logout', to: 'sessions#destroy'
+  
+  # post 'enroll_event', to: "users#enroll_event"
```
Comment (dhruvika29kevat) [inline config/routes.rb:80]: remove commented routes

**Review by dhruvika29kevat — COMMENTED** [review]
Which comment functionality is working ?
From comments controller or the methods defined in events controller?

**Comment by rutikpatel20** [comment]
I will remove all the commented code.

**Pull request merged**